### PR TITLE
Feature/education inbox update UI

### PR DESCRIPTION
**Pull request by FernandoBoza**
## Description

**Updates the Education Inbox to now Education Letters app name and path.**

**Updates for education/education-letters**
If user is visitor or logged in. If not logged in remain in page, else redirect to education/education-letters/preview

**Updates for education/education-letters/preview**
The user letters status page where users can either have a letter OR no letter. 
If a user can download a letter, they'll have the choice to download via a download button.

## Original issue(s)
[20229](https://github.com/department-of-veterans-affairs/vets-website/pull/20229)

## Testing done
- [ ] In progress

## Definition of done
- [X] Logged in users can see status of letters via a download  page or no letters page.
- [X] Non logged in users  text and copy for intro page has been updated

### CODE REVIEW

File: src/applications/education-letters/containers/App.jsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+            <b>Note:</b> If you have an older decision letter—or you’re a family
+            member or dependent—you can contact us through Ask VA to request a
+            copy of your letter.{' '}
+            <a href="https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fask.va.gov%2F&data=04%7C01%7Cherbert.anagho%40accenturefederal.com%7C5b0be35e33a2487d4a0c08d9ecb991bc%7C0ee6c63b4eab4748b74ad1dc22fc1a24%7C0%7C0%7C637801104030719343%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=QuGxWs9osAHjaGwInFjQO5cwEQ%2BK84u9J3XH2QcwZNk%3D&reserved=0">
+              Request your VA education letter through Ask VA.
+            </a>
+          </p>
+        </div>
+
+        <div style={{ marginTop: '3rem', marginBottom: '6.4rem' }}>
```
Comment (timwright12) [inline src/applications/education-letters/containers/App.jsx:98]: Can we use the design system utilities for this? https://design.va.gov/utilities/margins

File: src/applications/education-letters/containers/InboxPage.jsx [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+  const HasLetters = (
+    <>
+      <FormTitle title="Your VA education letters" />
+      <p className="va-introtext">
+        Download your VA education decision letters.
+      </p>
+      <h2>Letters available for you to download</h2>
+      <div className="edu-certi-eligibility">
+        <h3 style={{ marginTop: '1rem' }}>Education decision letter</h3>
```
Comment (timwright12) [inline src/applications/education-letters/containers/InboxPage.jsx:46]: Can we use design system utilities for this? https://design.va.gov/utilities/margins

File: src/applications/education-letters/containers/InboxPage.jsx [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+          percentage of benefits you’re entitled to, and the time limit for
+          using your benefits.
+        </p>
+        <div>
+          <a
+            className="vads-u-flex--1"
+            download
+            href="http://localhost:3000/meb_api/v0/claim_letter"
+          >
+            <i className="fa fa-download vads-u-display--inline-block vads-u-margin-right--1" />
```
Comment (timwright12) [inline src/applications/education-letters/containers/InboxPage.jsx:59]:
```suggestion
            <i className="fa fa-download vads-u-display--inline-block vads-u-margin-right--1" aria-hidden="true" />
```

**Review by timwright12 — DISMISSED** [review]

**Review by nathanielbuck — APPROVED** [review]

**Review by theodur — COMMENTED** [review]
I would also recommend updating the [changed-apps-build](https://github.com/department-of-veterans-affairs/vets-website/blob/master/config/changed-apps-build.json#L21) config to use the new entry name

**Review by theodur — APPROVED** [review]

**Comment by zeki-yaren** [comment]
Testing is still in progress

**Pull request merged**